nutils/sample.py

```diff
@@ -69,7 +69,7 @@ class Sample(types.Singleton):
 
   @staticmethod
   @types.apply_annotations
-  def new(transforms:types.tuple[transformseq.stricttransforms], points:types.strict[PointsSequence], index:types.tuple[types.frozenarray[int]]=None):
+  def new(transforms:types.tuple[transformseq.stricttransforms], points:types.strict[PointsSequence], index:types.tuple[numeric.wrapped]=None):
```

Comment:
A `dtype` check is removed here. Should the behaviour of `_frozenarraymeta[dtype]` be reintroduced?

Reply:
That check was supposed to go to the constructor, indeed, good catch.